swagger/public_v1/component_schemas/ProviderFilter.yml

```diff
@@ -2,6 +2,39 @@ type: object
 example: ""
 description: "This schema is used to search within collections to return more specific results."
 properties:
+  can_sponsor_skilled_worker_visa:
+    description: "Only return providers that can sponsor skilled worker visa."
+    type: boolean
+    example: true
+  can_sponsor_student_visa:
+    description: "Only return providers that can sponsor student visa."
```

Comment:
```suggestion
    description: "Only return providers that can sponsor a Student Visa."
```

Reply:
Done in https://github.com/DFE-Digital/publish-teacher-training/pull/3150/commits/fc81aef3dbef15ef4a799576b67adbd1d9f472b7